lore/species/tajaran/The Great Frost.md

```diff
@@ -0,0 +1,85 @@
+
+The Early Years
+
+In early years the people who lived within the hearth engines spent much of their time simply surviving as scavengers. Yet even as the first decade came to a close plans and actions were being taken. In those early times the tajaran began to organize greater works for survival. Progress was slow and rife with failures and hazards but they began to build technology powered by the hearth engines. The first early creations were the arboretums, the frost breakers, and the deadly weapon known as the Glare of S�randarr.
+
+Food was a rare resource in the frozen wastelands and with its scarcity the first problem to be solved was created. The light and heat of the hearth engines provided the necessity to form functioning hydroponics chambers. The problem was not the environment or even the technology, that they possessed, it was the lack of seeds from which to cultivate. Some holds contained seed vaults with which to grow from, yet others did not obtain them before the Great Frost happened and many were left unable to fix their food problem. This is where the Sun Wardens began to shift into true legend, as time went on scavenging became less and less important to the slowly thriving holds. These rough necks and frost-land veterans took it upon themselves to travel to holds who had seed vaults. They braved miles upon miles of snow and danger to gather seeds for their people. Few survived, but those that did brought salvation with them. With the food crisis in each hold fading away their attention turned to other problems.
+
+The next crisis to tackle became the lack of raw materials that were always in demand, lumber and iron were of great importance but no amount of scavenging could ever meet what was needed. As a means to an end they began to develop primitive yet sturdy machines in some holds. Robots of pure programming and no mind capable of operating longer than any man and in places they themselves could not go. These early machines were clunky but effective. Before the great frost the tajaran people were only just reaching the stage of true advancement in robotics. The ever praised Akari had equipped many holds with that they needed to continue this work, though few managed to create any more than two or three robots. Small in number or not, these machines were capable of working day and night performing simple tasks such as digging, mining, and the culling of trees. Over time with the revival of faith the position on what these tools meant varied among many, but some saw these machines as a gift from S�randarr and decorated them with religious symbols and carvings. In time, they were collectively called the frost breakers as their number and use swelled.

```

Comment:
"... always in demand. Lumber and iron..."
"As a means to an end, they began to develop primitive, yet sturdy machines..."
"Before the Great Frost, the tajaran..."
"... holds with **what** they needed to..."